dashboard/config/routes.rb

```diff
@@ -202,6 +202,9 @@ module OPS
   resources :followers, only: [:create]
   post '/followers/remove', to: 'followers#remove', as: 'remove_follower'
 
+  post '/followers/create_async', to: 'followers#create_async', defaults: {format: 'json'}
+  post '/followers/remove_async', to: 'followers#remove_async', defaults: {format: 'json'}
+
```

Comment:
Actually, the thing I'd really love to do here is extend the `sections` API that Josh and I [are porting over to dashboard right now](https://github.com/code-dot-org/code-dot-org/pull/16451), since your actions are always implicitly in the context of the current user and don't use follower IDs.  Just reading through this, the API URLs that feel right to me are

```
/api/v1/sections/<section_code>/join
/api/v1/sections/<section_code>/leave
```
Maybe we should pivot that direction instead?